.storybook/preview.tsx

```diff
@@ -16,6 +16,7 @@ import type { Preview, StoryFn } from '@storybook/react-webpack5';
 import React from 'react';
 
 import Theme from './Theme';
+import * as tokens from '../src/tokens-dist/ts/colors';
```

Comment:
Use the generated / named tokens that are sync'd to the style dictionary token file